doc/user_documentation.rst

```diff
@@ -859,6 +859,22 @@ contents of the directories.  Either please ignore the inconsistency, or visit
 the `non-cached host variant
 <http://copr-be.cloud.fedoraproject.org/results/>`_.
 
+.. _`faq-build-timeout`:
+
+.. rubric:: My build failed because of a timeout, why?  :ref:`¶ <faq-build-timeout>`
+
+Builds are not allowed to run forever. The default limit is 5 hours
+(18000 seconds) but users can increase it up to 30 hours (108000
+seconds). If the timeout limit is exceeded, the build will be killed
+with the following error message:
```

Comment:
The timeout is actually [50 hours](https://pagure.io/fedora-infra/ansible/blob/fa3499d30df60c1edd3ea8499f60c617548ced53/f/roles/copr/frontend/templates/copr.conf#_151-153).

But it is a deployment specific (I'm never sure what way to document these options, we should separate the Fedora Copr config from the generic Copr documentation in the future).  But in this case, I'd use some more generic statement, like:

The default timeout is typically 5h, and maximum timeout 50h.  But this is
deployment specific, e.g. take a look at [`MAX_BUILD_TIMEOUT` option for Fedora
Copr](https://pagure.io/fedora-infra/ansible/blob/fa3499d30df60c1edd3ea8499f60c617548ced53/f/roles/copr/frontend/templates/copr.conf).

Reply:
Updated, PTAL.

Considering your bigger picture goal, decoupling the defaults and Fedora Copr specific values in the whole documentation, I'd go with something like this:

![Screenshot_2023-12-06_06-35-16](https://github.com/fedora-copr/copr/assets/2151771/4b538a65-1d30-4966-90fa-dcd911a9cba5)

In the future, it will be easy to remove all the admonitions from our documentation and put them wherever we decide to have the Fedora Copr specifics.